src/lib/components/TokenSelect/NoTokensAvailableOnNetwork.tsx

```diff
@@ -0,0 +1,32 @@
+import { Trans } from '@lingui/macro'
+import { ColumnCenter } from 'components/Column'
+import { HelpCircle } from 'lib/icons'
+import styled, { ThemedText } from 'lib/theme'
+
+const HelpCircleIcon = styled(HelpCircle)`
+  height: 64px;
+  margin-bottom: 12px;
+  stroke: ${({ theme }) => theme.secondary};
+  width: 64px;
+`
+
+const Text = styled(ThemedText.Body1)`
+  text-align: center;
+`
+
+const Wrapper = styled(ColumnCenter)`
+  height: 80%;
+  justify-content: center;
+`
+
+function NoTokensAvailableOnNetwork() {
+  return (
+    <Wrapper>
```

Comment:
Use a `Column` instead of `Wrapper` to avoid custom styling. You can use gap to add the spacing, and you don't need to specify a height if it doesn't overflow.